guides/2022-04-05-conduit-kafka-connect-connector.mdx

```diff
@@ -0,0 +1,136 @@
+1. Create a Conduit Pipeline.
+1. Add the Connector to Pipeline.
+
+<!--truncate-->
+
+## Setup
+
+The Kafka Connect wrapper connector is written in Java so it needs to be compiled. 
+
+To begin, clone the connector:
+
+```
+git clone git@github.com:ConduitIO/conduit-kafka-connect-wrapper.git
+```
+
+Then, build the connector:
+
+```bash
+cd conduit-kafka-connect-wrapper
+
+./scripts/dist.sh
+```
+
+This script will:
+
+1. Create a directory called `dist`.
+1. Compile the connector and place the JAR in `dist`.
+1. Create `dist/libs` directory.
+
+Now that we have everything setup, we can add the Kafka Connect connectors. 
+
+## Adding Kafka Connect Connector
+
+The `libs` directory is where you put the Kafka Connect connector JARs and their dependencies (if any). The wrapper plugin will automatically load connectors and all the other dependencies from a `libs` directory.
+
+To make things easy, we have included [Aiven's Kafka Connect JDBC Connectors](https://github.com/aiven/jdbc-connector-for-apache-kafka). This connector allows you to connect to any [JDBC database](https://en.wikipedia.org/wiki/Java_Database_Connectivity#:~:text=Java%20Database%20Connectivity%20(JDBC)%20is,Edition%20platform%2C%20from%20Oracle%20Corporation.). 
+
+For this example, we will use the PostgreSQL Kafka Connect JDBC Connector. To install, add the following:
+
+- [Postgres Connector JAR](https://repo1.maven.org/maven2/org/postgresql/postgresql/42.3.3/postgresql-42.3.3.jar)
+
+## Adding Connector to Pipeline
+
+Now that our connector is setup, and with Kafka Connect connectors is included in `lib`, we can use the connector within a pipeline.
+
+1. [Start Conduit](https://github.com/ConduitIO/conduit#installation-guide).
+2. Create Pipeline: 
+
+```curl
+curl -X 'POST' \
+  'http://localhost:8080/v1/pipelines' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "config": {
+    "name": "my-pipeline",
+    "description": "This pipeline is for testing",
+  }
+}'
+```
+
+3. Add Connector to Pipeline:
+
+```curl
+curl -X 'POST' \
+  'http://localhost:8080/v1/connectors' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "type": "TYPE_SOURCE",
+  "plugin": "/path/to/conduit-kafka-connect-wrapper/dist/conduit-kafka-connect-wrapper",
+  "pipeline_id": "f24c4a70-8664-4f80-9c27-204825442943",
+  "config": {
+    "name": "my-pg-source",
+    "settings": {
+      "wrapper.connector.class": "io.aiven.connect.jdbc.JdbcSourceConnector",
+      "connection.url": "jdbc:postgresql://localhost/conduit-test-db",
+      "connection.user": "username",
+      "connection.password": "password",
+      "incrementing.column.name": "id",
+      "mode": "incrementing",
+      "tables": "customers",
+      "topic.prefix": "my_topic_prefix"
+    }
+  }
+}'
+```
+
+When creating a Conduit connector, the `plugin` you need to use is the path to `dist/conduit-kafka-connect-wrapper`. 
```

Comment:
Do we need `dist` here? Seems less confusing to say the plugin path is the path to `conduit-kafka-connect-wrapper` rather than including that extra bit.